src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -9130,6 +9186,61 @@ static void verify(CSharpCompilation comp)
             }
         }
 
+        [Fact]
+        public void LocalScope_04_For()
+        {
+            var source =
+@"for (scoped s1 = default;;) {
+for (ref scoped s2 = ref s1;;) {break;} // 1
```

Comment:
consider indenting to make the nesting clear. Also, it looks like there is no diagnostic given on this line.

Reply:
>  it looks like there is no diagnostic given on this line.

The same as in the test above. Comments aren't there to convey any expectations. They are  there to make it easier to map baseline to the source. Expectations are set by baselines.

Reply:
Will address indenting in the next PR